src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,111 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import './FindMovie.scss';
-
+import { getConnect } from '../../api/goods';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ onFindFilm }) => {
+  const [title, setTitle] = useState('');
+  const [film, setFilm] = useState('');
+
+  const findMovieHandler = (nameFilm) => {
+    if (nameFilm === '') {
+      return;
+    }
+
+    getConnect(nameFilm).then((response) => {
+      if (response.Response === 'False') {
+        setFilm('');
+
+        return;
+      }
+
+      setFilm({
+        title: response.Title,
+        description: response.Plot,
+        imgUrl: response.Poster,
+        imdbUrl: `https://www.imdb.com/title/${response.imdbID}/`,
+        imdbId: response.imdbID,
+      });
+
+      setTitle('');
+    });
+  };
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+  const addFilmHandler = () => {
+    if (!film) {
+      return;
+    }
+
+    onFindFilm(film);
+    setFilm('');
+  };
+
+  return (
+    <>
+      <form
+        className="find-movie"
+        onSubmit={(event) => {
+          event.preventDefault();
+          onFindFilm(film);
+        }}
+      >
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className="input is-danger"
```

Comment:
"is-danger" should appear here only when something goes wrong, now the red border around the input is visible all the time